app/src/main/java/com/example/courses/data/db/PersonData.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.courses.data.db
+
+import android.content.Context
+import androidx.room.Room
+import com.example.courses.data.repositories.PersonRepository
+import com.example.courses.domain.entity.Person
+import io.reactivex.Observable
+import kotlinx.coroutines.flow.Flow
+
+class PersonData(context: Context): PersonRepository {
+
+    private var database = Room.databaseBuilder(
+        context,
+        DbDatabase::class.java,
+        "personDatabase"
+    ).allowMainThreadQueries().build()
```

Comment:
1. It's reasonable to move each function call to the next line.
2. MainThread should not be allowed